tools/rostest/src/rostest/rostest_parent.py

```diff
@@ -45,17 +46,50 @@
 import roslaunch.parent
 
 from rosmaster.master import Master
+from rospy import logwarn
+
 
 class ROSTestLaunchParent(roslaunch.parent.ROSLaunchParent):
 
-    def __init__(self, config, roslaunch_files, port=0):
+    def __init__(self, config, roslaunch_files, port=0, reuse_master=False, clear=False):
         if config is None:
             raise Exception("config not initialized")
         # we generate a run_id for each test
-        run_id = roslaunch.core.generate_run_id()
+        if not reuse_master:
```

Comment:
Negating this conditional seems unnecessary. Wouldn't it read cleaner without an additional `not`?